src/net/tcp/listener.rs

```diff
@@ -41,6 +44,20 @@ pub struct TcpListener {
 }
 
 impl TcpListener {
+    /// Creates a new `TcpListener` from a standard `net::TcpListener`.
```

Comment:
Can you move the method below `bind`, to indicate in the documentation that `bind` is the preferred way to create a listener.